automathor.py

```
@@ -0,0 +1,205 @@
+        return f"{self.filename}:{self.line}\n{self.author_email} {self.datetime}\n{self.commit}\n"
+
+    def render(self) -> str:
+        """Render the context as a markdown file."""
+        template_loader = jinja2.FileSystemLoader(searchpath="./")
+        template_env = jinja2.Environment(loader=template_loader, autoescape=True)
+        template = template_env.get_template("template.jinja2")
+
+        output = template.render(context=self)
+        return output
+
+    def user(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `user`

Add return type annotation: `str`

(ANN201)